packages/ag-charts-community/src/util/ticks.ts

```diff
@@ -114,9 +114,8 @@ export function getTickInterval(
 
 export function tickStep(start: number, end: number, count: number, minCount = 0, maxCount = Infinity): number {
     if (start === end) {
-        return 1;
-    }
-    if (count < 1) {
+        return Math.min(Math.max(1, minCount), maxCount);
```

Comment:
```suggestion
        return clamp(1, minCount, maxCount);
```
nit: use `clamp`, it improved readability